### DISCUSSION TITLE
Netlify CMS Successor

### DISCUSSION THREAD

**Discussion by etmartinkazoo** [withastro/docs#2070] [Ideas]
Netlify CMS seems to be end of life: https://github.com/netlify/netlify-cms/discussions/6503

There is a fork, Static CMS, here: https://github.com/StaticJsCMS/static-cms

I think it is worth paying attention to but I was interested if people using Netlify CMS are not worried or if others have any experience with Static CMS. 

I think my favorite Git Based CMS is Tina.io (https://tina.io/) They just went version 1.0

**Comment by delucis** [comment]
I maintain [`astro-netlify-cms`](https://github.com/delucis/astro-netlify-cms/) and I’ve been keeping an eye on the Static CMS fork, curious to see how it pans out. I guess we have to wait and see.

**Reply by etmartinkazoo** [reply]
Hi @delucis thanks for responding. Is it incorrect to say Netlify CMS is not being maintained any more? I have a small software shop and would be happy to use Netlify CMS. I'm concerned it is going to be more or less abandoned.

Thank you for your work @delucis !

**Reply by delucis** [reply]
Don’t think it’s incorrect no — no-one at Netlify has explicitly said it, but 6 months without even merging dependency updates etc. suggests it may be abandoned. I’ve also been hesitant in investing more time in that Astro integration for that reason. If Static CMS is able to port over that good work and keep things going that would be awesome — there’s some nice concepts there!

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
